osbs/build/build_requestv2.py

```diff
@@ -86,6 +86,16 @@ def render(self, validate=True):
         data = self.get_reactor_config_data()
         self.set_data_from_reactor_config(data)
 
+        koji_task_id = self.user_params.koji_task_id.value
+        if koji_task_id is not None:
+            self.set_label('koji-task-id', str(koji_task_id))
+
+            # keep also original task for all manual builds with task
+            # that way when delegated task for autorebuild will be used
+            # we will still keep track of it
+            if self.triggered_after_koji_task is None:
```

Comment:
This is `BuildRequestV2` specific, attr `self.triggered_after_koji_task` is nto defined in this class

Reply:
right